OpenRA.Mods.Common/TraitsInterfaces.cs

```diff
@@ -39,6 +39,18 @@ public interface IPlaceBuildingDecorationInfo : ITraitInfo
 	}
 
 	[RequireExplicitImplementation]
+	public interface IBlocksProjectiles
+	{
+		WDist BlockingHeight { get; }
+	}
+
+	[RequireExplicitImplementation]
+	public interface IBlocksProjectilesInfo : ITraitInfo
+	{
+		WDist BlockingHeight { get; }
```

Comment:
`BlockingHeight` can change at runtime, so what does this represent? The maximum blocking height?  Lets rename this to make it clear.